package.json

```diff
@@ -60,6 +68,7 @@
     "protractor": "~4.0.14",
     "rimraf": "^2.5.4",
     "@types/node": "^6.0.46",
-    "@types/jasmine": "2.5.36"
+    "@types/jasmine": "2.5.36",
+    "D3": "4.9.1"
```

Comment:
This line causes `npm install` to fail (at least on Linux.. might not be failing on case insensitive platforms). Doesn't actually need to be there at all since nothing in this change set depends on D3.